pyproject.toml

```diff
@@ -57,6 +57,12 @@ dependencies = [
 milvus = ["pymilvus>=2.5.0"]
 mqtt = ["paho-mqtt>=2.1.0"]
 dns = ["dnspython>=2.8.0"]
+otel = [
+    "opentelemetry-sdk>=1.38.0",
+    "opentelemetry-exporter-otlp-proto-grpc>=1.38.0",
```

Comment:
should we also install http? if we do, we can also remove the slightly complicated handling of ImportError.

In general, we dont need to support an incomplete installation (missing urllib3 instrumentation or whatnot). Just crashing is cleaner.